src/docs/management-groups.md

```diff
@@ -0,0 +1,75 @@
+# Using Management Groups with Mission Landing Zone
+
+## Concepts
+
+### MLZ Structure
+
+The base Mission Landing Zone (MLZ) tiers (SACA Hub, Tier 0, Tier1, Tier 2) can be deployed across one or more Azure subscriptions.  The  mission workloads (Tier 3) can be deployed into separate subscriptions per workload, consilidated into a single subscription, or a combination of these approaches can be used based on your needs.  So from a management perspective, there is usually one SACA Hub subscription, one Tier 0 subscription, one Tier 1 subscription, one Tier 2 subscription and one or more Tier 3 subscriptions.  
```

Comment:
Change 'consilidated' to 'consolidated'. 😄

Reply:
Done